internal/inventory/config.go

```diff
@@ -30,17 +26,28 @@ type ApplicationSettings struct {
 	AdjustLastReadOnByOrigin bool
 }
 
-// WriteableConfig is a struct representation of the Writeable section of the configuration.toml file.
-type WriteableConfig struct {
-	LogLevel string
+// CustomConfig is the struct representation of all of the sections from the configuration.toml
+// that we are interested in syncing with Consul.
+type CustomConfig struct {
+	AppSettings ApplicationSettings
+	Aliases     map[string]string
 }
 
-// ConsulConfig is the struct representation of all of the sections from the configuration.toml
-// that we are interested in syncing with Consul.
-type ConsulConfig struct {
-	Writable            WriteableConfig
-	ApplicationSettings ApplicationSettings
-	Aliases             map[string]string
+type ServiceConfig struct {
+	AppCustom CustomConfig
+}
+
+// UpdateFromRaw updates the service's full configuration from raw data received from
+// the Service Provider.
+func (c *ServiceConfig) UpdateFromRaw(rawConfig interface{}) bool {
```

Comment:
You should add a note in the comment that this function implements the `UpdatableConfig` interface for `ServiceConfig`.

Reply:
added